### DISCUSSION TITLE
[Task]: Support for 2.5D

### DISCUSSION THREAD

**Discussion by DominikFilipiak** [open-edge-platform/anomalib#1204] [Feature Requests]
### What is the motivation for this task?

I'd like to have support for 2.5D images (like stacked images of the same object at different depths). This is a common use case in medical imaging.

### Describe the solution you'd like

Incorporation of backbones/architectures ready to work with 2.5D stacked images.

### Additional context

Maybe anyone knows about good workarounds/hacks for this?

**Comment by samet-akcay** [comment]
Hi @DominikFilipiak, thanks for proposing this. I personally don't have experience with this use-case. Can you please provide some more reference(potentially papers, repos or datasets?)

**Comment by DominikFilipiak** [comment]
Hi @samet-akcay I’ll try to provide something soon. Thanks for replying.

**Comment by DominikFilipiak** [comment]
Dear @samet-akcay sorry for not replying sooner. There are some papers on 2.5D in general computer vision [1], especially in medical segmentation [2,3]. Because of that, I was under the impression that there will be something for AD,  but after some search, I could not find anything like that. So that was my bad - sorry about that.

I asked because I'm dealing with such a problem (2.5D AD). I decided the easiest workaround is to train separate models for different Z-axis values. Works pretty well, but unfortunately, I can't share more details.

BTW I love the library, thanks to all the maintainers!

[1] https://ieeexplore.ieee.org/document/8803757
[2] https://openreview.net/pdf?id=Ob62JPB_CDF
[3] https://arxiv.org/pdf/1910.10398.pdf
